mobile/www/config.json

```diff
@@ -1,3 +1,3 @@
 {
-	"SERVER_URL": "put_server_url_here"
+	"SERVER_URL": "https://sda-vehicledam-3-29.mybluemix.net/"
 }
```

Comment:
did you mean to check this in? Or is this assumed to always be available?

Reply:
nope, that's a mistake